hips/tiles/tile.py

```diff
@@ -83,21 +79,19 @@ def nside(self) -> int:
         return hp.order2nside(self.order)
 
     @property
-    def dst(self) -> np.ndarray:
-        """Destination array for projective transform"""
-        return np.array(
-            [[self.tile_width - 1, 0],
-             [self.tile_width - 1, self.tile_width - 1],
-             [0, self.tile_width - 1],
-             [0, 0]],
-        )
-
-    @property
     def skycoord_corners(self) -> SkyCoord:
         """Corner values for a HiPS tile"""
         theta, phi = boundaries(self.nside, self.ipix)
         return SkyCoord(phi, np.pi / 2 - theta, unit='radian', frame=self.frame)
 
+    def dst(self, tile_width) -> np.ndarray:
```

Comment:
@tboch  and I suggest to move this to a property on the `SimpleHipsTilePainter` class (or a standalone function in the `draw.py` code).